x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/EnterpriseSearch.java

```diff
@@ -63,6 +72,7 @@
 
 public class EnterpriseSearch extends Plugin implements ActionPlugin, SystemIndexPlugin {
     public static final String ENGINE_API_ENDPOINT = "_engine";
+    public static final String BEHAVIORAL_ANALYTICS_API_ENDPOINT = "_behavioral_analytics";
```

Comment:
Just curious, did you consider `_analytics` or something similar, as it would be a shorter endpoint?

Reply:
The challenge is the url isn't part of the engine's path so  `HOST/_analytics` might be too generic. We also have an analytics module in elasticsearch so want the name to be more targeted to us.  I think `_analytics` makes sense if it was `HOST/_engine/<engine>/analytics`